Cargo.toml

```diff
@@ -6,13 +6,24 @@ description = "Simple package to count generic iterables"
 repository = "https://github.com/coriolinus/counter-rs"
 documentation = "https://docs.rs/counter"
 readme = "README.md"
-keywords = ["count", "multiset", "bag"]
+keywords = ["count", "multiset", "bag","counter"]
 license = "MIT"
-edition = "2018"
+edition = "2021"
 
 [dependencies]
 num-traits = "0.2"
 
 [dev-dependencies]
 maplit = "1.0"
 rand = "0.8.5"
+
+[profile.release]
+opt-level = 3
+debug = false
+debug-assertions = false
+overflow-checks = false
+lto = true
+panic = 'unwind'
+incremental = false
+codegen-units = 1
+rpath = false
```

Comment:
This is unnecessary, as this is library code which is only ever a dependency:

> Cargo only looks at the profile settings in the Cargo.toml manifest at the root of the workspace. Profile settings defined in dependencies will be ignored.

[docs](https://doc.rust-lang.org/cargo/reference/profiles.html#profiles)